waku/common/wakubridge.nim

```diff
@@ -51,15 +44,17 @@ type
 func toWakuMessage(env: Envelope): WakuMessage =
   # Translate a Waku v1 envelope to a Waku v2 message
   WakuMessage(payload: env.data,
-              contentTopic: ContentTopic(uint32.fromBytes(env.topic)),
+              contentTopic: ContentTopic(uint32.fromBytes(env.topic, Endianness.bigEndian)),
```

Comment:
Is this different from default behavior?

Reply:
Default is "system" endianness which on most systems will be little-endian. I decided on big-endian because a `contentTopic` of `1` translating to a `v1` `topic` of `[0,0,0,1]` feels slightly more intuitive than `[1,0,0,0]`.